### PR TITLE
refactor: update to monorepo structure

### PR DESCRIPTION
**Pull request by dolcalmi**
this is an split of https://github.com/GaloyMoney/price/pull/17. This PR migrates realtime service to a monorepo structure.

Most of the changes are configuration and file renaming (relocation)

### CODE REVIEW

File: realtime/src/servers/realtime/run-monitoring.ts [hunk 1]
```diff
@@ -44,7 +44,13 @@ for (const currency of supportedCurrencies) {
       observerResult.observe(price, { label: "median" })
 
       const exchangePrices = await Realtime.getExchangePrices(currency)
-      if (exchangePrices instanceof Error) return
+      if (exchangePrices instanceof Error) {
```
Comment (vindard) [inline realtime/src/servers/realtime/run-monitoring.ts:47]: Would place like this be good for new tracing events as well?

File: realtime/src/app/errors.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import * as DomainErrors from "@domain/errors"
 import * as ExchangesErrors from "@domain/exchanges/errors"
-import * as ProvidersErrors from "@domain/providers/errors"
 import * as PrimitivesErrors from "@domain/primitives/errors"
+import * as PriceErrors from "@domain/price/errors"
 
 export const ApplicationErrors = {
```
Comment (vindard) [inline realtime/src/app/errors.ts:6]: Are we adding an error-map switch to do any sort of mapping in this repo as well?

**Review by vindard — APPROVED** [review]
New layout looks sensible to me so far, can't spot any immediate red flags

image

**Review by bodymindarts — APPROVED** [review]

File: dev/otel-agent-config.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  jaeger:
+    protocols:
+      thrift_compact: # on port 6831
+      thrift_binary: # on port 6832
+
+processors:
+  batch:
+  attributes:
+    actions:
+    - key: apiKey
```
Comment (bodymindarts) [inline dev/otel-agent-config.yaml:11]: again the reminder don't forget to PR to prod config.

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**